src/pages/Bridge/BurnForm/index.tsx

```diff
@@ -209,9 +209,8 @@ const BurnForm = (): JSX.Element | null => {
           />
           <TokenField
             id={WRAPPED_TOKEN_AMOUNT}
-            name={WRAPPED_TOKEN_AMOUNT}
             label={WRAPPED_TOKEN_SYMBOL}
-            ref={register({
+            {...register(WRAPPED_TOKEN_AMOUNT, {
```

Comment:
This change was required for version 7. Migration guide here: https://react-hook-form.com/migrate-v6-to-v7/